hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/snapshot/OMSnapshotPurgeRequest.java

```diff
@@ -103,6 +103,8 @@ public OMClientResponse validateAndUpdateCache(OzoneManager ozoneManager,
             trxnLogIndex, updatedSnapInfos, true);
         updateSnapshotChainAndCache(omMetadataManager, fromSnapshot,
             trxnLogIndex, updatedPathPreviousAndGlobalSnapshots);
+        ozoneManager.getOmSnapshotManager().getSnapshotCache()
```

Comment:
Can you please add test for it? Could be as simple as checking cache size after the purge finishes.

Reply:
Yup! sure definitely! @hemantk-12 please take a look the new verification part added in [TestSnapshotDeletingService.java](https://github.com/apache/ozone/pull/5248/commits/f7a4b3886cd67fa7ba1275aca9a70197bdab88ff#diff-4467e2d5f908fe0f61428c3328bd0a0c797d0b07e3a38ec40969f2264277ecba)
Feel free to let me know if any question, thanks!

Reply:
Thanks @DaveTeng0